R/insert_cassette.R

```diff
@@ -31,7 +31,7 @@ insert_cassette <- function(
   }
   # enable webmockr
   webmockr::enable(quiet = vcr_c$quiet)
-  sup_mssg(vcr_c$quiet, webmockr::webmockr_allow_net_connect())
+  suppressMessages(webmockr::webmockr_allow_net_connect())
```

Comment:
Technically a behaviour change, but unlikely to affect many folks. I think probably we can make `vcr_c$quiet` go away in favour of better logging tools.